crates/cargo-test-support/src/git.rs

```diff
@@ -124,11 +124,31 @@ impl Repository {
 
 /// Initialize a new repository at the given path.
 pub fn init(path: &Path) -> git2::Repository {
+    default_search_path();
     let repo = t!(git2::Repository::init(path));
     default_repo_cfg(&repo);
     repo
 }
 
+fn default_search_path() {
+    use crate::paths::GLOBAL_ROOT;
+    use git2::{opts::set_search_path, ConfigLevel};
+    lazy_static::lazy_static! {
```

Comment:
FWIW instead of using `lazy_static` this can use a `sync::sync::Once`

Reply:
Fixed. Thanks!

This pull request is almost ready for review. Is there any plan to rollout a new release of git2?

Reply:
Sure yeah, I just published a new version.